build.gradle

```diff
@@ -44,6 +44,9 @@ signing {
     sign configurations.archives
 }
 
+def sonatypeUsername = System.getenv('sonatypeUsername')
```

Comment:
Yeah, when I tried to run it as is, I would get a compilation failure because gradle had no idea that these were defined - that meant that I couldn't even run a ./gradlew eclipse. An alternative would be to remove this here and check in a gradle.properties with dummy values